MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs

```diff
@@ -108,11 +108,28 @@ public TimeSpan Buffered
             get
             {
                 var buffered = TimeSpan.Zero;
-                if (Player.CurrentItem != null)
-                    buffered =
-                        TimeSpan.FromSeconds(
-                            Player.CurrentItem.LoadedTimeRanges.Select(
-                                tr => tr.CMTimeRangeValue.Start.Seconds + tr.CMTimeRangeValue.Duration.Seconds).Max());
+
```

Comment:
I don't think I fully understand, how would you approach simplifying this getter?